src/main/java/model/card/CardDeck.java

```diff
@@ -0,0 +1,64 @@
+package model.card;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Queue;
+import java.util.stream.Stream;
+
+public class CardDeck {
+
+    private static final int MIN_DECK_COUNT = 1;
+
+    private final Queue<Card> deck;
+
+    public CardDeck(Queue<Card> deck) {
+        this.deck = deck;
+    }
+
+    public static CardDeck createShuffledDeck(int deckCount) {
+        validateDeckCount(deckCount);
+        List<Card> deck = generateDeck(deckCount);
+        Collections.shuffle(deck);
+        return new CardDeck(new ArrayDeque<>(deck));
+    }
+
+    private static void validateDeckCount(int deckCount) {
+        if (deckCount < MIN_DECK_COUNT) {
+            throw new IllegalArgumentException("1개 이상의 덱을 사용해야 합니다.");
+        }
+    }
+
+    private static List<Card> generateDeck(int deckCount) {
+        List<Card> deck = new ArrayList<>();
+        for (int count = 0; count < deckCount; count++) {
```

Comment:
```suggestion
        for (int i = 0; i < deckCount; i++) {
```
개인적인 취향차이일 수 있지만 for문의 i를 임의로 변경하면 
가독성이 굉장히 안좋아지는것 같아요
변경없이 처리하는건 어떨까요?

Reply:
애매한 부분이었는데, 확실히 말해주셔서 감사합니다😊